app/config/config.yml

```diff
@@ -137,7 +137,7 @@ liip_theme:
     themes:
         - baggy
         - material
-    autodetect_theme: wallabag_core.helper.detect_active_theme
+    # autodetect_theme: wallabag_core.helper.detect_active_theme
```

Comment:
just a test, because wallabag tried to load the active theme when I did some tests. I will enable it soon.